performance/server/src/main/java/org/hawkular/apm/performance/server/Service.java

```diff
@@ -140,7 +141,7 @@ public void setRegistry(ServiceRegistry registry) {
      * @param btxnName The optional business txn name
      */
     public void call(Message mesg, String interactionId, String btxnName) {
-        boolean activated = collector.activate(uri, null, interactionId);
+        boolean activated = collector.session().activate(uri, null, interactionId);
```

Comment:
The test was changed to use the TraceCollector instead of the impl DefaultTraceCollector - the interface does not directly expose this operation, it is on the SessionManager interface which DefaultTraceCollector happens to also implement.